### PR TITLE
Fixes Keywords not showing up and authors not saving

### LINKED ISSUE #290: [Item Record] Metadata not showing on item record page
When uploading content, metadata appears inconsistently

### LINKED ISSUE #292: [Item Record] Authors are not being displayed after ingest
I loaded one Excel file and one Word document. Upon editing these documents after upload, the authors do not appear in the form fields

### LINKED ISSUE #293: [Item Record] Rights statement is not being saved

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #290 Fixes #292 Fixes #293

### CODE REVIEW

File: app/forms/file_edit_form.rb [hunk 1]
```diff
@@ -26,6 +26,20 @@ def self.build_permitted_params
...
+    permitted << { :date_created => [] }
+    permitted << { :identifier => [] }
+    permitted << { :based_near => [] }
+    permitted << { :related_url => [] }
+    permitted << { :language => [] } 
+    permitted << { :publisher => [] }
+    permitted << { :spatial => [] }
+    permitted << { :provenance => [] }
+    permitted << { :date => [] }
+    permitted << { :doi => [] }
```
Comment (straleyb) [inline app/forms/file_edit_form.rb]: I wanted to push this up just so we had it but I dont think we need all these permitted attributes.

File: app/helpers/sufia_helper.rb [hunk 2]
```diff
@@ -24,6 +24,8 @@ def link_to_field(fieldname, fieldvalue, displayvalue = nil)
     link_url = catalog_index_path(p)
     if fieldvalue["preflabel"]
       display = displayvalue.blank? ? fieldvalue["preflabel"] : displayvalue
+    elsif fieldvalue["id"]
```
Comment (straleyb) [inline app/helpers/sufia_helper.rb]: This is an extra check for the way licenses work

**Comment by straleyb** [comment]
@revgum @wickr @luisgreg99 this branch should be good to go.

**Comment by luisgreg99** [comment]
@straleyb: It looks good to me, I tested it out locally and I can confirm that it's saving the data we need

**Pull request merged**